subsys/zbus/Kconfig

```diff
@@ -37,6 +37,10 @@ config ZBUS_MSG_SUBSCRIBER_BUF_ALLOC_STATIC
 
 endchoice
 
+config ZBUS_MSG_SUBSCRIBER_NET_BUF_POOL_ISOLATION
+	default n
+	bool "Channels can use the global pool or a separated one."
```

Comment:
```suggestion
	bool "Enable isolated pool instead of only using the global pool."
```

Reply:
Done. I have used the text: "Use isolated pools instead of only using the global pool." based on the rule of not using "enable" for prompts: https://docs.zephyrproject.org/latest/build/kconfig/tips.html#prompt-strings